### ISSUE TITLE
Base url

### ISSUE THREAD

**Issue by damc** [niklasbuschmann/contrast#30]
My blog is in some folder, so the url of my blog is:

https://xyz.com/something

instead of just https://xyz.com

The problem is that all of the links on the website lead to https://xyz.com/post_name instead of https://xyz.com/something/post_name . For that reason CSS and links don't work.

I tried to set  "baseUrl" config to '/something', but that didn't work. It works in onther themes like "Minima" for example.

Can I get this theme to work with a blog in a folder? What is the best way to do that?

**Comment by niklasbuschmann** [comment]
This should work exactly the same with as with "Minima". Both themes use the `relative_url` [liquid filter](https://jekyllrb.com/docs/liquid/filters/).

Have you tried using `baseurl` instead of `baseUrl`?

**Comment by damc** [comment]
Yes, you're right. I should have used ``baseurl`` instead of ``baseUrl``.
